internal/jet/literal_expression.go

```diff
@@ -160,6 +161,66 @@ func Decimal(value string) FloatExpression {
 	return &floatLiteral
 }
 
+// ---------------------------------------------------//
+
+type boolArrayLiteral struct {
+	arrayInterfaceImpl[BoolExpression]
+	literalExpressionImpl
+}
+
+func BoolArray(values []bool) ArrayExpression[BoolExpression] {
+	l := boolArrayLiteral{}
+	l.literalExpressionImpl = *literal(pq.BoolArray(values))
+	l.arrayInterfaceImpl.parent = &l
+
+	return &l
+}
+
+type integerArrayLiteral struct {
+	arrayInterfaceImpl[IntegerExpression]
+	literalExpressionImpl
+}
+
+func Int64Array(values []int64) ArrayExpression[IntegerExpression] {
```

Comment:
Can be done without new literal types. Some of these files need some cleaning.

```golang
func Int64Array(values []int64) ArrayExpression[IntegerExpression] {
    return ArrayExp[IntegerExpression](literal(pq.Int64Array(values))
}
```

Reply:
Cleanedup